index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Mobile First Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <style> @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap'); </style>
+  <style> @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap'); </style>
+  <style> @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap'); </style>
+</head>
+<body>
+  <header>
+    <div class="toolbar">
+      <a href="https://github.com/bilalrajput09" class="name">
+        Bilal Ahmed
+      </a>
+        <img src="C:\Users\HP\My-Portfolio\img\threelines.png">
+    </div>  
+  </header>
+  <div class="introduction">
+    <h1>Hey There.
+      I’m Bilal Ahmed</h1>
```

Comment:
- [ ] I see 😃that you didn't add the background image for section one, so kindly remember to add the background image to be like the design.💯

| Figma design | Your design| 
| --- | --- |
|   ![image](https://user-images.githubusercontent.com/83360636/212332188-86bfbd18-dd4b-481d-9640-23d1d7982139.png) |    ![image](https://user-images.githubusercontent.com/83360636/212332303-1111da68-ccf6-47c4-8ab9-4696783821fc.png)|